src/components/QuranReader/ReadingView/Line.tsx

```diff
@@ -23,9 +25,17 @@ export type LineProps = {
   quranReaderStyles: QuranReaderStyles;
   pageIndex: number;
   lineIndex: number;
+  isFixedWidth?: boolean;
```

Comment:
`Line` is only used by the ReadingView I think we don't need this prop and always apply the styling `isFixedWidth`